packages/cli/src/commands/org/config/pull.ts

```diff
@@ -31,15 +39,49 @@ export default class Pull extends Command {
       multiple: true,
       description: 'The resources types to pull from the organization.',
     }) as IOptionFlag<(keyof typeof ResourceType)[]>,
+    snapshotId: flags.string({
+      char: 's',
+      exclusive: ['resourceTypes'],
+      description:
+        'The unique identifier of the snapshot to pull. If not specified, a new snapshot will be created.',
+    }),
   };
 
   public static hidden = true;
 
   @Preconditions(IsAuthenticated())
   public async run() {
+    cli.action.start('Creating Snapshot');
     const snapshot = await this.getSnapshot();
+
+    cli.action.start('Updating project with Snapshot');
     await this.refreshProject(snapshot);
+
     await snapshot.delete();
+    cli.action.stop('Project updated');
+  }
+
+  public async catch(err?: Error) {
+    const {flags} = this.parse(Pull);
+    await this.config.runHook(
+      'analytics',
+      buildAnalyticsFailureHook(this, flags, err)
+    );
+    handleSnapshotError(err);
+    this.displayAdditionalErrorMessage(err);
+  }
+
+  private displayAdditionalErrorMessage(err?: Error) {
+    if (err instanceof SnapshotOperationTimeoutError) {
+      const snapshot = err.snapshot;
+      cli.info(
+        dedent`
+
+          Once the snapshot is created, you can pull it with the following command:
+
+              ${blueBright`coveo org:config:pull ${snapshot.id}`}`
```

Comment:
wow thats lot of spacing innit?

Reply:
I can reduce it to 2 spaces

Reply:
> I can reduce it to 2 spaces

can you show some screenshot for current and with just 2 space (I'm talking about the breaklines btw more than the spacing itself, sorry was unclear)

Reply:
![image](https://user-images.githubusercontent.com/12199712/125810369-18227fef-ed86-4415-891c-554ff2ba8a0c.png)

VS 

![image](https://user-images.githubusercontent.com/12199712/125810132-fb4041d4-05a6-49b7-a29a-9f3dc5c26a30.png)